airbyte-integrations/connector-templates/generator/build.gradle

```diff
@@ -14,7 +14,7 @@ task testScaffoldTemplates
 
 def addScaffoldTemplateTask(name, packageName,scaffoldParams=[]) {
     def taskName = "testScaffoldTemplate_${name}"
-    def outputDir = "airbyte-integrations/connectors/source-scaffold-${name}"
+    def outputDir = "airbyte-integrations/connectors/source-${name}"
```

Comment:
I don't believe this was previously working:

```
WARNING: Error loading config file: .dockercfg: $HOME is not defined
> Configure project :airbyte-cdk:python

> Task :airbyte-integrations:connector-templates:generator:testScaffoldTemplate_Python HTTP API Source
Scaffolding connector

> core-js-pure@3.6.5 postinstall /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connector-templates/generator/node_modules/core-js-pure
> node -e "try{require('./postinstall')}catch(e){}"

added 297 packages from 243 contributors and audited 298 packages in 2.933s

15 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


> airbyte-connector-generator@0.1.0 generate /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connector-templates/generator
> plop "Python HTTP API Source" "scaffold-source-http"

- 
✖  +! File already exists
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/Dockerfile
- 
✖  ++ ../../connectors/source-{{dashCase name}}/.dockerignore Aborted due to previous action failure
- 
✖  emitSuccess Aborted due to previous action failure
- 

> Task :airbyte-integrations:connector-templates:generator:testScaffoldTemplate_Python Source
Scaffolding connector
audited 298 packages in 1.26s

15 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


> airbyte-connector-generator@0.1.0 generate /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connector-templates/generator
> plop "Python Source" "scaffold-source-python"

- 
✖  +! File already exists
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-python/Dockerfile
- 
✖  ++ ../../connectors/source-{{dashCase name}}/.gitignore Aborted due to previous action failure
- 
✖  ++ ../../connectors/source-{{dashCase name}}/.dockerignore Aborted due to previous action failure
- 
✖  emitSuccess Aborted due to previous action failure
- 
```

Reply:
okay this is working now 
```
> airbyte-connector-generator@0.1.0 generate /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connector-templates/generator
> plop "Python HTTP API Source" "scaffold-source-http"

- 
✔  +! 23 files added
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/Dockerfile
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/README.md
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/acceptance-test-config.yml
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/acceptance-test-docker.sh
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/build.gradle
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/main.py
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/requirements.txt
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/setup.py
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/integration_tests/__init__.py
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/integration_tests/abnormal_state.json
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/integration_tests/acceptance.py
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/integration_tests/catalog.json
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/integration_tests/configured_catalog.json
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/integration_tests/invalid_config.json
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/integration_tests/sample_state.json
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/secrets/config.json
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/source_scaffold_source_http/__init__.py
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/source_scaffold_source_http/source.py
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/source_scaffold_source_http/spec.json
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/unit_tests/unit_test.py
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/source_scaffold_source_http/schemas/TODO.md
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/source_scaffold_source_http/schemas/customers.json
 -> /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/source_scaffold_source_http/schemas/employees.json
- 
✔  ++ /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http/.dockerignore
- 
✔  emitSuccess
- 

🚀 🚀 🚀 🚀 🚀 🚀

Success! 

Your scaffold-source-http connector has been created at /actions-runner/_work/airbyte/airbyte/airbyte-integrations/connectors/source-scaffold-source-http.
```

Reply:
although I think the caching here is broken; problem for another day

Reply:
@davinchia Did this fail even when you rm rf'd the directory beforehand on your local? I think this failed because the scaffold template already existed on your local machine. CI deletes this directory before running so it doesn't fail. I think we should keep it the way it is since it seems like it was working as intended. WDYT?

Reply:
these logs are from the CI - I don't believe this was working for a long time 😬